<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigateWindow</class>
 <widget class="QWidget" name="NavigateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>15</height>
          </size>
         </property>
         <property name="value">
          <number>30</number>
         </property>
         <property name="format">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBack">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="firstNaviText">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="secondNaviText">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="thirdNaviText">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnRecoil">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Recoil</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDone">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
